src/userGuide/UserGuideFi.tsx

```diff
@@ -547,26 +543,26 @@ function UserGuideFi(): ReactElement {
       </UserGuideAccordion>
       <h2 id="_Tietojen_poisto">Tietojen poisto</h2>
       <p>
-        Voit poistaa tietosi joko yksittäisistä palveluista tai koko profiilin.
-        Poiston myötä kaikki tietosi palvelusta poistetaan tai anonymisoidaan,
-        jos palvelu esim. lakisääteisenä joutuu niitä säilyttämään. Sinulla ei
-        kuitenkaan ole poiston jälkeen pääsyä tietoihin, eikä niitä ole
-        yhdistettävissä sinuun.
+        Voit poistaa tietosi yksittäisistä palveluista tai halutessasi koko
+        profiilin. Poiston myötä kaikki tietosi palvelusta poistetaan tai
+        anonymisoidaan, jos palvelu esim. lakisääteisenä joutuu niitä
+        säilyttämään. Sinulla ei kuitenkaan ole poiston jälkeen pääsyä
+        tietoihin, eikä niitä ole yhdistettävissä sinuun.
```

Comment:
```suggestion
        tietoihin, eivätkä ne ole yhdistettävissä sinuun.
```

Reply:
Tai vaihtoehtoisesti
```suggestion
        tietoihin, eikä niitä voi yhdistää sinuun.
```

Reply:
Lisään jälkimmäisen.